engines/bastion_katello/app/assets/javascripts/bastion_katello/common/views/subscription-add-or-remove.html

```diff
@@ -21,7 +21,7 @@
        </td>
      </tr>
      <tr class="grey-table-row" bst-table-row ng-repeat-end ng-repeat="subscription in subscriptions" row-select="subscription">
-       <td bst-table-cell>
+       <td bst-table-cell ng-hide="!contextAdd">
```

Comment:
I think the double negative here (hide  and !) would be clearer with just `ng-show="contextAdd"`